lib/ansible/modules/cloud/amazon/elb_target_group.py

```diff
@@ -130,6 +130,19 @@
     description:
       - The identifier of the virtual private cloud (VPC). Required when I(state) is C(present).
     required: false
+  wait:
+    description:
+      - Indicates if the module has to wait to the target group to be created.
+    required: false
+    default: yes
+    choices: [ 'yes', 'no' ]
+    version_added: 2.6
```

Comment:
This isn't true though. We need a way of overriding `version_added` for documentation that got missed (we now have tests for that to prevent it happening in future). We should remove `version_added` as I'm pretty sure it's the same as the module's `version_added`.